frontend-next-migration/src/preparedPages/MainPage/ui/_components/sections/PlayWithUs/PlayWithUs.module.scss

```diff
@@ -125,6 +125,35 @@
   margin-top: 2rem;
 }
 
+.WebGLNotice {
+  position: absolute;
```

Comment:
consider to avoid absolute positioning. This has a negative impact on accessibility, and also does not always work adaptively. And I think it is better to remove the fixed height for the component and do it through margin (in this case, absolute breaks everything...)
![image](https://github.com/user-attachments/assets/1e2fcae9-9e04-459e-8f86-5cc5193df3f7)